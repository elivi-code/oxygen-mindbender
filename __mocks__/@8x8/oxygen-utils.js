export * from '@8x8/oxygen-utils';

const useFontSize = jest.fn(() => 16);

export { useFontSize };
