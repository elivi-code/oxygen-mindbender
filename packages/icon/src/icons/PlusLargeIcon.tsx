import React from 'react';
import type { IconProps } from '../utils/createSvgIcon';
import createSvgIcon from '../utils/createSvgIcon';

export type { IconProps };
export default createSvgIcon(<g><path fillRule="evenodd" d="M12 2c-.5523 0-1 .4477-1 1v8H3c-.5523 0-1 .4477-1 1s.4477 1 1 1h8v8c0 .5523.4477 1 1 1s1-.4477 1-1v-8h8c.5523 0 1-.4477 1-1s-.4477-1-1-1h-8V3c0-.5523-.4477-1-1-1Z" clipRule="evenodd"/></g>, 'PlusLargeIcon');
