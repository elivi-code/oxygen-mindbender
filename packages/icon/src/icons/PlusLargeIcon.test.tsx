/* eslint-disable */
import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { icon as theme } from '@8x8/oxygen-constants';
import PlusLargeIcon from './PlusLargeIcon';

describe('<PlusLargeIcon>', () => {
  let wrapper;

  function setup(props = {}) {
    return renderer.create(<PlusLargeIcon theme={theme} {...props} />).toJSON();
  }

  it('should render correctly', () => {
    wrapper = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
/* eslint-enable */
