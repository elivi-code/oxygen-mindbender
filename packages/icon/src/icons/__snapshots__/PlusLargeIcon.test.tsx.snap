// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PlusLargeIcon> should render correctly 1`] = `
.c0 {
  height: 24px;
  width: 24px;
  min-height: 24px;
  min-width: 24px;
  -webkit-user-select: auto;
  -moz-user-select: auto;
  -ms-user-select: auto;
  user-select: auto;
  pointer-events: auto;
  fill: #1C2025;
}

<svg
  className="c0"
  color={null}
  disabled={false}
  role="presentation"
  selected={false}
  size={null}
  viewBox="0 0 24 24"
>
  <g>
    <path
      clipRule="evenodd"
      d="M12 2c-.5523 0-1 .4477-1 1v8H3c-.5523 0-1 .4477-1 1s.4477 1 1 1h8v8c0 .5523.4477 1 1 1s1-.4477 1-1v-8h8c.5523 0 1-.4477 1-1s-.4477-1-1-1h-8V3c0-.5523-.4477-1-1-1Z"
      fillRule="evenodd"
    />
  </g>
</svg>
`;
